### PR TITLE
DDF-1326: Sets the id property along with shortname

### PR DESCRIPTION
**Pull request by vinamartin**
@stustison 
@andrewkfiedler 
@djblue

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf-catalog/111)

### CODE REVIEW

**Comment by coyotesqrl** [comment]
Since this is in the area of one of my open PRs I thought I should take a quick look... 

:+1:

File: admin/module/catalog-admin-module-sources/src/main/webapp/js/view/ModalSource.view.js [hunk 1]
```diff
@@ -236,6 +236,8 @@ function (ich,Marionette,Backbone,ConfigurationEdit,Service,Utils,wreqr,_,$,moda
             if (!_.isUndefined(config)) {
                 var properties =  config.get('properties');
                 properties.set('shortname', name);
+                properties.set('id', name);
```
Comment (andrew-fiedler) [inline admin/module/catalog-admin-module-sources/src/main/webapp/js/view/ModalSource.view.js]: If you pass an object into the set method, you can set multiple properties at the same time.    

````javascript
properties.set({
       'shortname': name,
       'id': name
});
````

Other than that, :+1:

Comment (stustison) [inline admin/module/catalog-admin-module-sources/src/main/webapp/js/view/ModalSource.view.js]: I forgot about that! Nice!

Comment (vinamartin) [inline admin/module/catalog-admin-module-sources/src/main/webapp/js/view/ModalSource.view.js]: Changed! Thanks! Didn't know this was possible! :smile:

**Comment by djblue** [comment]
:+1:

**Pull request merged**